### DISCUSSION TITLE
Questions about the DigitalOcean in student packets.

### DISCUSSION THREAD

**Discussion by newbe3p** [community/community#68042] [GitHub Education]
### Hi everyone,

Hi everyone,I passed the student package verification, but after I used Digital Ocean's discount, my Digital Ocean account was banned. What should I do?
Please help me.

**Comment by kamlendras** [comment]
Thank You,
Report it at https://support.github.com/contact/education.
MARK THIS ANSWER AS ANSWERED :)
